var/spack/repos/builtin/packages/mvapich-plus/package.py

```diff
@@ -0,0 +1,365 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import itertools
+import os.path
+import re
+import sys
+
+from spack.package import *
+
+
+class MvapichPlus(AutotoolsPackage):
+    """Mvapich is a High-Performance MPI Library for clusters with diverse
+    networks (InfiniBand, Omni-Path, Ethernet/iWARP, and RoCE) and computing
+    platforms (x86 (Intel and AMD), ARM and OpenPOWER)"""
+
+    homepage = "https://mvapich.cse.ohio-state.edu/userguide/userguide_spack/"
+    url = "https://mvapich.cse.ohio-state.edu/download/mvapich/mv2/mvapich-3.0.tar.gz"
+    list_url = "https://mvapich.cse.ohio-state.edu/downloads/"
+
+    maintainers("natshineman", "harisubramoni", "MatthewLieber")
+
+    executables = ["^mpiname$", "^mpichversion$"]
+
+    license("Unlicense")
```

Comment:
Small nitpick, but I'd switch the ordering of the directives and executables variable. Directives should be placed after `executables` in order to group variables before directives.